composer.json

```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.0|^8.1"
+        "php": "~7.4.0|~8.0.0|~8.1.0|~8.2.0"
```

Comment:
what about `7.4 - 8.2`? https://getcomposer.org/doc/articles/versions.md#hyphenated-version-range-

Reply:
maybe is less used but is the same and shorter

Reply:
Thank you @noelamaya. Good to know that;-D. In this case, I think it's preferable to use the more standard and verbose way, cost you can see in a single watch which PHP versions are supported.